includes/data-stores/class-wc-customer-download-log-data-store.php

```diff
@@ -0,0 +1,196 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * WC Customer Download Log Data Store.
+ *
+ * @version  3.3.0
+ * @category Class
+ * @author   WooThemes
+ */
+class WC_Customer_Download_Log_Data_Store implements WC_Customer_Download_Log_Data_Store_Interface {
+
+	/**
+	 * Create download log entry.
+	 *
+	 * @param WC_Customer_Download_Log $download_log
+	 */
+	public function create( &$download_log ) {
```

Comment:
Should we add a type hint for this param? (And do the same for the other methods in this class)